mlir-tensorrt/compiler/lib/Conversion/TensorRTToTensorRTRuntime/TensorRTToTensorRTRuntime.cpp

```diff
@@ -57,6 +57,80 @@ struct RewriteConstants : public OpRewritePattern<tensorrt::ConstantOp> {
   }
 };
 
+// Helper function to convert both CallOp and CallAllocOp
+static LogicalResult
+convertCallOp(Operation *op, MLIRContext *ctx, IRRewriter &rewriter,
```

Comment:
Don't include `MLIRContext` in the argument list.... the context can be grabbed from any of the other arguments